pyproject.toml

```diff
@@ -145,7 +145,6 @@ ignore = [
     "B007", # Loop control variable `i` not used within loop body
     "B009", # Do not call `getattr` with a constant attribute value
     "B010", # [*] Do not call `setattr` with a constant attribute value.
-    "B011", # Do not `assert False` (`python -O` removes these calls)
```

Comment:
I'm in favor of ignoring this rule, this is pytest we're talking about, the mother of all asserts :)

Reply:
Yeah, of course, it broke the test I should have seen that coming !

Reply:
Dropped the commit in latest push.